<?xml version="1.0" encoding="utf-8"?>
<launch>
  <arg name="robot_id" default="" />
  <arg name="robot_ns" default="multirotor$(arg robot_id)" />
  <arg name="shift_x" default="0.0"/>
  <arg name="shift_y" default="0.0"/>

  <node pkg="forest_navigation" type="acc_check.py" name="acc_check" output="screen" ns="$(arg robot_ns)" >
    <param name="shift_x" type="double" value="$(arg shift_x)" />
    <param name="shift_y" type="double" value="$(arg shift_y)" />
    <param name="robot_ns" value="$(arg robot_ns)" />
  </node>
</launch>